Dockerfile

```diff
@@ -117,6 +117,7 @@ RUN pip install -r requirements/edx/base.txt
 
 # Install node and node modules
 RUN nodeenv /edx/app/edxapp/nodeenv --node=16.14.0 --prebuilt
+COPY scripts/copy-node-modules.sh scripts
```

Comment:
Is it weird that we're copying node modules (and presumably installing them above) before we've installed the right version of npm?

Reply:
@davidjoy it's not weird, but I can certainly see why it reads weirdly. I've shifted this statement to be after `npm install npm` (it's arbitrary, but I think it reads better now) and I've added a comment explaining the statement must come before `npm ci`. Let me know if that doesn't makes things clearer!

Reply:
OH.  Duh, I didn't read carefully enough that that was a `COPY`. I get it.  👍🏻